### ISSUE TITLE
Feature Request: Theming app: add logo-wide

### ISSUE THREAD

**Issue by ungesundes-halbwissen** [nextcloud/server#338]
With the new theming app introduced in 9.0.52 you can specify a custom logo displayed on the login screen (upper middle) AND on the Nextcloud pages themselves in the top left corner.

As the login page logo tends to be a more squared size than the "pages" logo, it would be nice to have the option to specify dedicated logos for login page and browsing pages.

**Comment by LukasReschke** [comment]
@nextcloud/designers Thoughts?

**Comment by jancborchardt** [comment]
We phased out logo-wide some time ago in favor of only showing a logo and text next to it. App name when logged in, and »Nextcloud« (or branded name) for share links etc. Including logo-wide will make things a lot more complicated since the positioning of these things would need to be adjusted according to the width. Also it most probably will break things on mobile.

So as said, we phased out logo-wide for several reasons.
